integration/kubernetes/k8s-expose-ip.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Saves the ip addresses
+declare -a IP_ADDRESSES
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	deployment="hello-world"
+	service="my-service"
+	port="8080"
+	image="gcr.io/google-samples/node-hello:1.0"
+	get_pod_config_dir
+	sudo -E crictl pull "$image"
+}
+
+@test "Expose IP Address" {
+	# Create deployment
+	kubectl create -f "${pod_config_dir}/deployment-expose-ip.yaml"
+
+	# Check deployment creation
+	kubectl wait --for=condition=Available deployment/${deployment}
+
+	# Get deployment
+	kubectl get deployments ${deployment}
+
+	# Describe deployment
+	kubectl describe deployments ${deployment}
+
+	# Display information about ReplicaSet objects
+	kubectl get replicasets
+	kubectl describe replicasets
+
+	# Expose deployment
+	kubectl expose deployment/${deployment} --type=LoadBalancer --name=${service}
+
+	# Get service
+	kubectl get services ${service}
+	kubectl describe services ${service}
+
+	# Verify pods IP Addresses
+	IP_ADDRESSES=$(kubectl get pods --output=wide | awk '{print $6}' | tail -n +2)
```

Comment:
Maybe we can do something more specific here using the `get pods -o ` JSON formatting, something like (but not, this is just a hacked up notion....):
```
kubectl get pods -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
```

Reply:
@grahamwhaley , thanks for the suggestion, I changed it to
```
kubectl get pods -o jsonpath='{.items[*].status.podIP}'
```